NEWS.md

```diff
@@ -7,6 +7,8 @@ shiny 1.3.2.9000
 
 ### Improvements
 
+* Resolved [#612](https://github.com/rstudio/shiny/issues/612): An optional paramter `alt` can be passed to `renderPlot()` and `renderCachedPlot()`, which is then added to the img-tag.
+
```

Comment:
It would be clearer to write this as: ``added to the `img` tag``.  Or you could use `<img> tag` (with the brackets).

Also, at the end, there should be a link to this PR.